README.md

```diff
@@ -35,6 +35,7 @@ At a high level an operator using the SDK processes events for watched resources
 First, checkout and install the operator-sdk CLI:
 
 ```sh
+$ mkdir -p $GOPATH/src/github.com/operator-framework/operator-sdk
 $ cd $GOPATH/src/github.com/operator-framework/operator-sdk
```

Comment:
@Comradin I think what's confusing is the missing `git clone` step. We can add that in to be more explicit.
So can you please replace lines 38-39 with:
```sh
$ mkdir -p $GOPATH/src/github.com/operator-framework
$ cd $GOPATH/src/github.com/operator-framework
$ git clone github.com/operator-framework/operator-sdk
$ cd operator-sdk
```